src/documentation/gtfs_to_ntfs_specs.md

```diff
@@ -187,7 +189,7 @@ immediately with an error.
 | trips.txt | route_id | Required | trips.txt | route_id | cf. NTFS `route_id` definition above to specify the proper reference. |
 | trips.txt | service_id | Required | trips.txt | service_id |  |
 | trips.txt | trip_id | Required | trips.txt | trip_id |  |
-| trips.txt | trip_headsign | Optional | trips.txt |  | `trip_short_name`, of if empty `trip_headsign` |
+| trips.txt | trip_headsign | Optional | trips.txt |  | `trip_short_name`, or if empty `trip_headsign` |
```

Comment:
Since NTFS 0.10 there are now 2 fields "trip_headsign" and "trip_short_name"

Reply:
I'm ok to do this later to avoid opening toi many subjects at once